Dockerfile

```diff
@@ -0,0 +1,23 @@
+#
+# Build
+#
+FROM maven:3.9.6-amazoncorretto-17-al2023@sha256:459be099faa25a32c06cd45ed1ef2bc9dbbf8a5414da4e72349459a1bb4d6166 AS buildtime
```

Comment:
we are building using gradle, not maven!
Moreover, because we are building using the wrapped gradle, we don't need to use an image dedicated to gradle: you could use the same image used for the Docker RUNTIME